Sources/GRPC/ConnectionManagerChannelProvider.swift

```diff
@@ -117,25 +119,42 @@ internal struct DefaultChannelProvider: ConnectionManagerChannelProvider {
     let hostname = self.serverHostname
     let needsZeroLengthWriteWorkaround = self.requiresZeroLengthWorkaround(eventLoop: eventLoop)
 
-    let bootstrap = PlatformSupport.makeClientBootstrap(group: eventLoop, logger: logger)
+    let bootstrap = PlatformSupport.makeClientBootstrap(
+      group: eventLoop,
+      tlsConfiguration: self.tlsConfiguration,
+      logger: logger
+    )
+
+    _ = bootstrap
```

Comment:
While I appreciate that our builder doesn't _actually_ require you follow the builder pattern, it would be _much_ less surprising if you actually chained off it (i.e. make `bootstrap` `var` instead of `let` and then assign it again here.